Bugzilla/API/V1/Github.pm

```diff
@@ -339,14 +338,18 @@ sub push_comment {
         }
       }
 
-      # Update the milestone to the nightly branch if closing the bug.
+      # Update the milestone to the nightly branch if default branch
       # Currently tailored for mozilla-mobile/firefox-android only
-      $self->_set_nightly_milestone($bug, $branch) if $repository eq 'mozilla-mobile/firefox-android';
-    }
+      if ($ref =~ /refs\/heads\/$default_branch/) {
```

Comment:
Nit: you could move the `$repository eq 'mozilla-mobile/firefox-android'` conditional into this one instead of having two separate `if` checks.

Reply:
Good point. Not sure why I didnt see that before :)